tools/generate_test_coverage_report.sh

```diff
@@ -17,4 +17,4 @@
 # It is also recommended to activate traces with ```RUST_LOG=debug``` before you generate the report.
 # This way the trace report also includes trace lines.
 # Without activated traces, the report complains that the trace lines are not covered by any test.
-RUST_LOG=debug cargo llvm-cov nextest --ignore-filename-regex "/main.rs|/test_utils.rs|/cli.rs" "$@"
+RUST_LOG=debug cargo llvm-cov nextest --ignore-filename-regex "/main.rs|/test_utils.rs|/cli.rs|/objects/agent_map.rs" "$@"
```

Comment:
It would be good if we could split the line somehow, s.t. we can write explanations why certain files are ignored.

Reply:
Done. You can have a look.